.vib/thanos/bucketweb/cypress/cypress/integration/thanos_bucketweb_spec.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+it('allows to see runtime & build information', () => {
+    cy.visit('/');
+    cy.contains('.dropdown-toggle', 'Status').click();
+    cy.contains('[class="dropdown-item"]', 'Runtime & Build Information').click();
+    cy.contains('Runtime Information').should('be.visible');
+    cy.get('h2').contains('Build Information').should('be.visible');
```

Comment:
```suggestion
    cy.contains('h2', 'Build Information').should('be.visible');
```

I think you can chain the selectors here. Ref: [doc](https://docs.cypress.io/api/commands/contains#Specify-a-selector-to-return-a-specific-element)